doc/source/development.rst

```diff
@@ -0,0 +1,77 @@
+Development Tips
+================
+
+If you are doing development on the Ray codebase, the following tips may be
+helpful.
+
+1. **Speeding up compilation:** Be sure to install Ray with
+
+   .. code-block:: shell
+
+     cd ray/python
+     python setup.py develop
```

Comment:
--user

Reply:
No strong pref (although personally I would not use that flag because conda takes care of that for me). However, people recommend using `pip install -e .` instead of `setup.py`, see https://stackoverflow.com/a/19048754